### PR TITLE
doc improvement: style fixes in Other samples

### PR DESCRIPTION
**Pull request by richabp**
Fixed inconsistencies in style and structure in
UART, Radio Test, BH1749, and Hello World samples.

Ref: NCSDK-13404

Signed-off-by: Richa Pandey <richa.pandey@nordicsemi.no>

### CODE REVIEW

**Review by kapi-no — APPROVED** [review]

File: samples/peripheral/radio_test/README.rst [hunk 1]
```diff
@@ -150,7 +150,7 @@ Building and running
 Testing
 =======
 
-After programming the sample to your development kit, you can test it in one of two ways.
+After programming the sample to your development kit, test it by performing the folowing steps in one of two ways:
```
Comment (peknis) [inline samples/peripheral/radio_test/README.rst:153]:
```suggestion
After programming the sample to your development kit, complete the following steps to test it in one of the following two ways:
```

File: samples/peripheral/radio_test/README.rst [hunk 2]
```diff
@@ -164,7 +164,7 @@ Testing with another development kit
    The kits are assigned a COM port (Windows) or ttyACM device (Linux), which is visible in the Device Manager.
 #. |connect_terminal_both|
 #. Run the following commands on one of the kits:
```
Comment (peknis) [inline samples/peripheral/radio_test/README.rst:166]: You also need an empty line before starting another list within a list.

File: samples/sensor/bh1749/README.rst [hunk 3]
```diff
@@ -37,6 +36,8 @@ The example below uses the Thingy:91.
 Sample Output
```
Comment (peknis) [inline samples/sensor/bh1749/README.rst:36]:
```suggestion
Sample output
```

File: samples/peripheral/radio_test/README.rst [hunk 4]
```diff
@@ -70,6 +53,23 @@ Skyworks front-end module
 
 You can configure the Skyworks front-end module (FEM) antenna output and activation delay using the main shell commands of the :ref:`radio_test_ui`.
 
+Overview
+********
+
+You can run the tests by connecting to the development kit through the serial port and sending shell commands.
```
Comment (peknis) [inline samples/peripheral/radio_test/README.rst:59]:
```suggestion
To run the tests, connect to the development kit through the serial port and send shell commands.
```

File: samples/peripheral/radio_test/README.rst [hunk 5]
```diff
@@ -70,6 +53,23 @@ Skyworks front-end module
...
+Zephyr's :ref:`zephyr:shell_api` module is used to handle the commands.
+At any time during the tests, you can dynamically set the radio parameters, such as output power, bit rate, and channel.
+In sweep mode, you can set the time for which the radio scans each channel from 1 millisecond to 99 milliseconds, in steps of 1 millisecond.
+The sample also allows you to send a data pattern to another development kit.
+
+The sample starts with enabling the high frequency crystal oscillator and configuring the shell.
```
Comment (peknis) [inline samples/peripheral/radio_test/README.rst:65]:
```suggestion
The sample first enables the high frequency crystal oscillator and configures the shell.
```

File: samples/peripheral/lpuart/README.rst [hunk 6]
```diff
@@ -31,6 +25,12 @@ The sample also requires the following pins to be shorted:
 
 Additionally, it requires a logic analyzer.
 
+Overview
+********
+
+The sample implements a simple loopback using a single UART instance.
+It has console and logging disabled by default, to demonstrate low power consumption while having UART active.
```
Comment (peknis) [inline samples/peripheral/lpuart/README.rst:32]:
```suggestion
By default, the console and logging are disabled to demonstrate low power consumption when UART is active.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Pull request merged**